public/static/docs/command-reference/import.md

```diff
@@ -15,17 +15,17 @@ import.
 usage: dvc import [-h] [-q | -v] [-o [OUT]] [--rev [REV]] url path
 
 positional arguments:
-  url         URL of Git repository with DVC project to download from.
+  url         URL of DVC or Git repository to download from.
   path        Path to data within DVC repository.
 ```
 
 ## Description
 
 DVC provides an easy way to reuse datasets, intermediate results, ML models, or
-other files and directories tracked in another <abbr>DVC repository</abbr> into
-the workspace. The `dvc import` command downloads such a <abbr>data
-artifact</abbr> in a way that it is tracked with DVC, so it can be updated when
-the data source changes. (See `dvc update`.)
+other files and directories tracked in another Git or <abbr>DVC
+repository</abbr> into the workspace. The `dvc import` command downloads such a
+<abbr>data artifact</abbr> in a way that it is tracked with DVC, so it can be
+updated when the data source changes. (See `dvc update`.)
```

Comment:
We can also base it on https://dvc.org/doc/command-reference/get#description:

`Provides an easy way to reuse datasets, intermediate results, ML models, or other files and directories tracked in any <abbr>DVC repository</abbr>, both by Git (e.g. source code) or DVC (e.g. datasets, ML models). The `dvc import` command downloads such a file or <abbr>artifact</abbr> in a way that it is tracked with DVC, so it can be updated when the data source changes. (See `dvc update`.)`

Reply:
I changed the wording here; what do you think of the new wording now?

Reply:
Thanks. Just one more comment: https://github.com/iterative/dvc.org/pull/900#pullrequestreview-339476045